sources/packages/web/src/composables/useModalDialog.ts

```diff
@@ -5,6 +5,7 @@ export function useModalDialog<T>() {
   let promise: (value: T) => void;
 
   const resolvePromise = (value: T) => {
+    showDialog.value = false;

```

Comment:
Thanks for doing this ❤️ 
@guru-aot @dheepak-aot @henryoforeh-aot, we no longer need to change the showDialog while consuming the modal.